frontend/src/index.scss

```diff
@@ -230,3 +236,26 @@ textarea::placeholder {
 .text-overflow-ellipsis {
   text-overflow: ellipsis;
 }
+
+@media (min-width: 64em) {
```

Comment:
It seems this block of code already exists in the same file between the lines 182-203 as follows:
https://github.com/aws-solutions/performance-dashboard-on-aws/blob/cceb061e6014a1696ab4b44af964835bd376f7fd/frontend/src/index.scss#L182-L203